site-graph/cache/cachemap/map.go

```diff
@@ -0,0 +1,36 @@
+package cachemap
+
+import (
+	"sync"
+
+	"github.com/adamplansky/go-bridge-mentoring/site-graph/cache"
+)
+
+// cmap implements cache interface and use golang map as underlying
+// data struct for storing objects
+
+type cachemap struct {
```

Comment:
imho: it is not worth implementing universal cache with interface{} if it is not shared library. Try to avoid common programmer temptation to make everything universal and DRY - this way you loose static typing and make everything more complex.
If you find that you copy-pasted something more than twice only then it worth considering something more universal and reusable.